tests/behat/features/web/translation/translate_project.feature

```diff
@@ -7,29 +7,112 @@ Feature: Project title, description and credits should be translatable via a but
       | 1  | Catrobat  |
     And there are projects:
       | id | name     | owned by | description   | credit   |
-      | 1  | project1 | Catrobat | mydescription |          |
-      | 2  | project2 | Catrobat |               | mycredit |
-      | 3  | project3 | Catrobat |               |          |
+      | 1  | project1 | Catrobat |               |          |
+      | 2  | project2 | Catrobat | mydescription |          |
+      | 3  | project3 | Catrobat |               | mycredit |
+      | 4  | project4 | Catrobat | mydescription | mycredit |
 
-  Scenario: Translate button should translate title and description when available
+  Scenario: Translate button should translate title when available
     Given I am on "/app/project/1"
     And I wait for the page to be loaded
-    Then the element "#translate-program" should exist
-    And the element "#translate-program" should have a attribute "href" with value "https://translate.google.com/?q="
-    And the element "#translate-program" should have a attribute "href" with value "project1"
-    And the element "#translate-program" should have a attribute "href" with value "mydescription"
+    Then the element "#program-translation-button" should exist
+    When I click "#program-translation-button"
+    And I wait for AJAX to finish
+    Then the element "#remove-program-translation-button" should be visible
+    And the element "#name-translation" should be visible
+    And the "#name-translation" element should contain "translated project1"
+    When I click "#remove-program-translation-button"
+    Then the element "#program-translation-button" should be visible
+    And the element "#name" should be visible

```

Comment:
Add
`And the "#name" element should contain "project1"`

Reply:
Updated